### ISSUE TITLE
quests

### ISSUE THREAD

**Issue by recurLi1** [zdzdliu/PPGArrhythmiaDetection#2]
However, the sklearn.metrics module does not contain a function called classification. Can you tell me if "classification" is a function I wrote myself? Can you provide me with the code?

**Comment by zdzdliu** [comment]
Refer to [in python, spyder, i import sklearn, why it doesn't find 'classification' in 'sklearn.metric'?](https://stackoverflow.com/questions/68498883/in-python-spyder-i-import-sklearn-why-it-doesnt-find-classification-in-sk)

You may try the following code
`from sklearn.metrics import classification_report, accuracy_score, confusion_matrix`

**Comment by bieyl** [comment]
Hello! Does the code work correctly? Thanks! @recurLi1 @zdzdliu
